lib/Backends/OpenCL/kernels.cl

```diff
@@ -328,11 +442,18 @@ size_t getNCHW(ShapeNCHW s, cl_uint32_t n, cl_uint32_t c, cl_uint32_t h,
 
 DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementadd, float, LHS + RHS)
 DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementsub, float, LHS - RHS)
-DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementmul, float, LHS *RHS)
+DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementmul, float, LHS * RHS)
 DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementdiv, float, LHS / RHS)
 DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementmax, float, max(LHS, RHS))
 DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementmin, float, min(LHS, RHS))
 
+DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL_QUANTIZED(elementadd, LHS + RHS)
+DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL_QUANTIZED(elementsub, LHS - RHS)
+DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL_QUANTIZED_M(elementmul, LHS * RHS)
```

Comment:
Could you group these macro-calls in such a way that we first call all non `_M` macros and then all `_M` macros?

Reply:
done